README.rst

```diff
@@ -234,7 +234,7 @@ command:
 Parse for XMLRunner errors
 --------------------------
 
-When you run [XMLRunner](https://github.com/xmlrunner/unittest-xml-reporting)
+When you run XMLRunner_,
 the errors are reported on the output, but they are not marked as failures in
 the test reports xml files. Since command exits as 1, we could not detect tests
```

Comment:
> Since command exits as 1

Should this be updated?

Reply:
No, xmlrunner command still exits as 1 and that is why we need warnings plugin...